sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -142,4 +143,14 @@ public sealed partial class DynamicData : System.Dynamic.IDynamicMetaObjectProvi
         System.Dynamic.DynamicMetaObject System.Dynamic.IDynamicMetaObjectProvider.GetMetaObject(System.Linq.Expressions.Expression parameter) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class DynamicDataOptions
+    {
+        public DynamicDataOptions() { }
+        public Azure.Core.Dynamic.DynamicDateTimeHandling DateTimeHandling { get { throw null; } set { } }
+    }
+    public enum DynamicDateTimeHandling
```

Comment:
If we expect that this enum will expand, does it make sense for it to be extensible?

Reply:
I believe extensible enums are intended for use with service enums that can change, and we would want the better UX of a CLR enum for this case.  IIRC adding a value to a CLR enum is a non-breaking change as long as the library version changes.